src/components/composer/composer/composer.ts

```diff
@@ -172,9 +173,16 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
         // render left
         assistantStyle += `right:0px;`;
       }
-      return (assistantStyle += `width:${ASSISTANT_WIDTH}px;`);
+      return assistantStyle;
+    }
+
+    const { width } = this.env.model.getters.getSheetViewDimensionWithHeaders();
```

Comment:
One of the main goal of the task was to have the assisstant work for for a composer in a side panel (https://www.odoo.com/odoo/2328/tasks/4045550?cids=1), in which case using `getSheetViewDimensionWithHeaders` won't work

Reply:
And it's breaking if a side panel is open

Reply:
Oops, I'm completely unfamiliar with this case. I'll put it back in dev and try to add a more generic fix. Thanks!